File: template/java/snip-getter-setter.java
/**
 * get {{_input_:name}}
 * @return {{_var_:name}}
 */
public {{_input_:type}} get{{_expr_:substitute('{{_var_:name}}', '\w\+', '\u\0', '')}}() {
	return {{_var_:name}};
}

/**
 * set {{_var_:name}}
 * @param {{_var_:name}}
 */
public void set{{_expr_:substitute('{{_var_:name}}', '\w\+', '\u\0', '')}}({{_var_:type}} {{_var_:name}}) {
	this.{{_var_:name}} = {{_var_:name}};
}

File: template/php/snip-getter-setter.php
/**
 * Get {{_input_:property}}
 *
 * @return {{_input_:type}}
 */
public function get{{_expr_:substitute('{{_var_:property}}', '\w\+', '\u\0', '')}}() {
	return $this->{{_var_:property}};
}

/**
 * Set {{_var_:property}}
 *
 * @param {{_var_:type}} {{_var_:property}}
 */
public function set{{_expr_:substitute('{{_var_:property}}', '\w\+', '\u\0', '')}}(${{_var_:property}}) {
	$this->{{_var_:property}} = ${{_var_:property}};
}
